FE Exercise/pass-bakery-fe-exercise-8/pass-bakery-fe.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add this iml file to your gitignore.

Reply:
Added the file to my gitignore